content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst

```diff
@@ -2,59 +2,70 @@
+.. image:: sub_assemblies/semifinished_product_bom.png
+   :align: center
+   :alt: A bill of materials for a semi-finished product.
 
-.. image:: sub_assemblies/sf_1.png
-    :align: center
+Create the top-level Bill of Materials (BoM)
+============================================
 
-Configure the Main BoM
-======================
+After the semi-finished products are fully configured, navigate to :menuselection:`Manufacturing
+--> Products --> Products`. Then, :guilabel:`Create` the top-level product. Configure the product’s
+specifications as desired, and be sure to :guilabel:`Save`.
 
-Then on the final product form, simply add your semi-finished 
-products to the Bill of Material. 
+Once the top-level product is configured, click the :guilabel:`Bill of Materials` smart button on
+the product form, then click :guilabel:`Create` to make a :abbr:`BoM (Bill of Materials)` for the
+top-level product. Then, simply add the semi-finished products to this :abbr:`BoM (Bill of
+Materials)`, along with any other necessary components.
 
-.. image:: sub_assemblies/sf_2.png
-    :align: center
+.. image:: sub_assemblies/custom_computer_bom.png
+   :align: center
+   :alt: A bill of materials for a top-level product, containing a subassembly component.
 
-Manage your production planning 
-=================================
+Manage production planning
+==========================
 
-There are several methods to manage the triggering of the various manufacturing orders. 
+There are several methods to manage manufacturing order automation for products with multilevel
+:abbr:`BoMs (Bills of Materials)`.
 
-If every time a manufacturing order is confirmed for the main product, you'd like one for 
-the semi-finished products as well, you have two options: 
+.. note:: 
+    Semi-finished products are specifically used to manage manufacturable products with multilevel
+    BoMs. If a BoM is being created simply to organize components or bundle sellable products,
+    using :doc:`Kits <kit_shipping>` is the more appropriate option.
 
-Option 1 : Create re-ordering rules for the semi-finished products, with both the minimum 
-and maximum desired stock quantities at 0. 
+To automatically trigger manufacturing orders for semi-finished products after confirming a
+manufacturing order for the main product, there are two options:
 
-.. image:: sub_assemblies/sf_3.png
-    :align: center
+**Option 1 (recommended):** Create *Reordering Rules* for the semi-finished products and set both
+the minimum and maximum desired stock quantities to '0'. To review how to set up reordering rules,
+refer to :doc:`this page <../../../inventory_and_mrp/purchase/products/reordering>`.
 
-Option 2 : Use the Replenish on Order (MTO) route on the semi-finished product, as well as 
-the manufacturing one. 
+**Option 2:** Activate the *Replenish on Order (MTO)* and *Manufacture* routes under the
+:guilabel:`Inventory` tab of the semi-finished product’s product form.
```

Comment:
Number of formatting suggestions here:
- Would first, format this as a bulleted list
- fix the apostrophe
- change '0' to `0` since it's input
- add `:guilabel:`'s to **Option 2** since these are checkboxes on the UI and they are being directly addressed to activate.

As well the `To review how to...` sentence might serve better as a `.. :seealso:` here, just like the one in the previous comment. Another option is to place an accumulated list of related documentation articles either at the beginning of the doc so the reader gets a heads up to be comfortable with those topics or at least have them open in a new tab. Up to you 🙃 

```suggestion
- **Option 1 (recommended):** Create *Reordering Rules* for the semi-finished products and set both
  the minimum and maximum desired stock quantities to `0`.

  .. seealso::
     - :doc:`Don't run out of stock with reordering rules
       <../../../inventory_and_mrp/purchase/products/reordering>`

- **Option 2:** Activate the :guilabel:`Replenish on Order (MTO)` and :guilabel:`Manufacture`
  routes under the :guilabel:`Inventory` tab of the semi-finished product's product form.
```